CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,87 @@
+Contributor Covenant Code of Conduct
+====================================
+
+Our Pledge
+**********
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our
```

Comment:
```suggestion
contributors and maintainers pledge to make participation in our
```

Reply:
This is the original text.. Shall we stick to that?

Reply:
ok, new version does not have this anymore ;)